scripts/strava_auth.py

```diff
@@ -119,9 +116,9 @@ def main(port, client_id, client_secret):
                         action='store', type=int, default=8000)
 
     parser.add_argument('--client-id', help='Strava API Client ID',
-                        action='store', default='28117')
+                        action='store', default='65392')
```

Comment:
I do have to change that since I need to use my own API because the IP on redirect will not be the localhost. I will change that to read it from a file..

Reply:
You can change in your local version, just don't push to the main repo. We set a default value so users who don't want to create their own app, can use ours.

Reply:
ok.. I will change it locally.

Reply:
done